File: packages/client/ios/RNOta/Resolver/BundleSource.swift
import Foundation

/// Where the resolved JavaScript bundle comes from.
public enum BundleSource: String, Sendable, Equatable {
  /// Ship-with-binary `main.jsbundle` — host uses its default URL.
  case embedded
  /// On-disk OTA slot (`bundle.hbc`).
  case ota
}

File: packages/client/android/src/main/java/com/rnota/resolver/BundleSource.kt
package com.rnota.resolver

/**
 * Where the resolved JavaScript bundle comes from.
 */
enum class BundleSource {
  /** Ship-with-binary asset / main.jsbundle — RN host uses its default path. */
  EMBEDDED,

  /** On-disk OTA slot (`bundle.hbc`). */
  OTA,
}
